docs/alerts/creating-logs-alerts.md

```diff
@@ -30,6 +30,18 @@ If you wanted to avoid using a specific threshold value and instead get notified
 
 ![image alt text](../images/alerts/image_6.png)
 
+# Transformation
+
+Transformations are used to modify metrics using mathematical expressions and functions. For example, imagine you want to get alerted when both `request.size` and `response.size` together exceed some threshold or become anomalous.  You would then transform them into a single dataseries by using an expression like this:
+
+```
+request.size + response.size
+```
+
+And then you would create an alert on this new data series.
+
+Read about [Transformations](../dashboards/chart-builder/#transformation) to leadn more about transformations, functions, and expressions used to perform transformations.
+
 Notice that the chart changed to help you get a sense of what would constitute an anomalous value in your case (dots outside the gray *confidence interval*). 
```

Comment:
Should this go before transformation section - it is referring to the image above?